src/pybind/mgr/CMakeLists.txt

```diff
@@ -22,4 +22,12 @@ install(DIRECTORY
   REGEX "rook/rook-client-python.*" EXCLUDE
   REGEX "osd_perf_query/.*" EXCLUDE
   REGEX "tox.ini" EXCLUDE
-  REGEX "requirements.txt" EXCLUDE)
+  REGEX "requirements.*\.txt" EXCLUDE
+  REGEX "constraints.*\.txt" EXCLUDE
+  REGEX "node_modules" EXCLUDE
+  REGEX "cypress.*" EXCLUDE
+  REGEX "\.coveragerc" EXCLUDE
+  REGEX "\.editorconfig" EXCLUDE
+  REGEX "\..*lintrc" EXCLUDE
+  REGEX "\.browserslistrc" EXCLUDE
+  REGEX "\.prettier*" EXCLUDE)
```

Comment:
@a16bitsysop @epuertat if this addresses the out-of-space issue, i am wondering if a follow-up PR which enumerates 

- `src/pybind/mgr/*.py` (exclude `hello`)
- `src/pybind/mgr/**` which contains `__init__.py` at its root directory.
- `dist/**`

would be more explicit. what do you think?

Reply:
Okay, do you mean all the python files are not needed either in mgr?

I found the alpine builders do have different free spaces, the aarch64 one has 

```
>>> Host system information (arch: aarch64, release: edge) <<<
- Number of Cores: 32
- Memory: 62.6868 Gb
- Free space: 33.2G
```

I am going to try and build locally as well

Reply:
no, i want to create a PR to use the "include" list instead of "exclude" list, once yours is merged. so that only the python files and the files under `dist` are installed.

Reply:
Ok, yes that makes sense and more future proof.

Reply:
> no, i want to create a PR to use the "include" list instead of "exclude" list, once yours is merged. so that only the python files and the files under dist are installed.

@tchaikov that was my initial idea here, but I later discarded  due to the risk of inadvertently missing files, since that case wouldn't be detected by regular Pull Request CI, but at the later Teuthology package-based integration testing.

In any case, your list makes sense. We would also need to include `src/pybind/mgr/dashboard/frontend/package.json`, which is used to get the default locale for the dashboard i18n.

Reply:
#42255